core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -442,9 +443,13 @@ object SparkEnv extends Logging {
     val addedJarsAndFiles = (addedJars ++ addedFiles).map((_, "Added By User"))
     val classPaths = (addedJarsAndFiles ++ classPathEntries).sorted
 
+    // Add Hadoop properties, it will ignore configs including in Spark
+    val hadoopProperties = hadoopConf.asScala.filter(entry => !conf.contains("spark.hadoop." + entry.getKey)).
```

Comment:
Why do you exclude "spark.hadoop" conf? it could be posible that they are showing in spark properties?

Reply:
Yes, these properties have shown above. So I remove it in hadoop properties.